proposals/NNNN-noncopyable-generics.md

```diff
@@ -417,11 +339,11 @@ protocol ArcadeCoin: Token /* , Copyable */ {}
 protocol CasinoChip: Token, ~Copyable {}
 ```
 
-A key takeaway from the example above is that `~Copyable` as a constraint is not viral:
-a protocol that has no `Copyable` constraint does _not_ mean its conformers cannot have the `Copyable` capability.
-As a corollary, a protocol that inherits from a noncopyable one can still be `Copyable`.
+Note that `~Copyable` as a constraint is not viral:
+types that conform to a `~Copyable` protocol can themselves be `Copyable`.
+Similarly, a protocol that inherits from a noncopyable one can still be `Copyable`.
```

Comment:
Clarifying that it's not protocols themselves that are copyable, but may or may not _require_ Copyable. Existentials constrained by even a noncopyable protocol will also be Copyable unless you opt-out with `~Copyable`.

```suggestion
Note that `~Copyable` as a constraint is not viral:
types that conform to a `~Copyable` protocol can themselves be `Copyable`.
Similarly, a protocol can still require its conformers to be `Copyable` even if it inherits
from a `~Copyable` protocol.
```

Reply:
I think it is covered in the existentials section.